server/http.go

```diff
@@ -135,7 +135,7 @@ func (p *Plugin) completeUserOAuthToZoom(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	zoomUser, authErr := client.GetUser(user)
+	zoomUser, authErr := client.GetUser(user, true)
```

Comment:
@mickmister 0/5 but it is usually a good practice to add a comment - smth. like `.GetUser(user, true /* first connect */ - using here the C/C# style of comments)` and `.GetUser(user, false /* not the first connect */)` to provide more context to the occasional code reader.

Reply:
@catalintomai I prefer to name the variable in this case, what do you think?

```suggestion
	firstConnect := true
	zoomUser, authErr := client.GetUser(user, firstConnect)
```

Reply:
Or `firstTimeConnecting` to be more explicit

Reply:
@mickmister, it is your call - it is a bit of a tradeoff between verbosity and laconism, have to check if golang allows you to have inline assignment e.g. smth like .`GetUser(user, firstConnect = true)`. I've definitely seen version I've mentioned above used countless times though in C# (not sure about golang).

Reply:
I've now gone with the variable name strategy I proposed above